docs/scos/dev/setup/quickstart-guides-install-spryker/quickstart-guide-install-spryker-on-macos-and-linux.md

```diff
@@ -62,24 +62,21 @@ docker/sdk bootstrap deploy.dev.yml
 
 ## 2. Update the hosts
 
-Update the hosts file using the command provided in the output of the previous step.
+Update the hosts file using the command provided in the output of the previous step. It should be similar to the following:
+
+```bash
+Please, run the following commands in order to prepare the environment:
```

Comment:
Do you don't think that the pic will be better? 

We only want to indicate what to expect as a successful response. But as you said they shouldn't try to copy it over.